lang/Français.json

```
@@ -1,160 +1,161 @@
+            "allowed_channels": "Chaînes autorisées :",
+            "all_channels": "Toutes",
             "and_more": "et {amount} de plus...",
             "percent_progress": "{percent} de {minutes} minutes",
             "minutes_progress": "{minutes} minutes"
         },
         "settings": {
             "general": {
-                "name": "Ajustements généraux",
-                "autostart": "Exécuter au démarrage du système: ",
-                "tray": "Exécuter dans la barre d'état système: ",
-                "priority_only": "Miner uniquement les jeux préférés: ",
-                "proxy": "Proxy (nécessite un redémarrage):"
+                "name": "Général",
+                "autostart": "Démarrage automatique :",
+                "tray": "Démarrage automatique dans la barre des tâches :",
+                "tray_notifications": "Notifications dans la barre des tâches :",
+                "priority_only": "Priorité uniquement :",
+                "proxy": "Proxy (nécessite un redémarrage) :"
             },
             "game_name": "Nom du jeu",
-            "priority": "Liste des jeux favoris",
-            "exclude": "Liste des jeux exclus",
+            "priority": "Priorité",
+            "exclude": "Exclure",
             "reload": "Recharger",
-            "reload_text": "La plupart des modifications nécessitent un rechargement ou un redémarrage de l'application pour prendre effet:"
+            "reload_text": "La plupart des modifications nécessitent un rechargement pour prendre effet immédiatement : "
         },
         "help": {
             "links": {
                 "name": "Liens utiles",
                 "inventory": "Afficher l'inventaire Twitch",
                 "campaigns": "Afficher toutes les campagnes et gérer les comptes associés"
             },
-            "how_it_works": "Comment ça marche",
-            "how_it_works_text": "Toutes les 60 secondes environ, l'application envoie un \"Minute vue\" à la chaîne actuellement regardée. \nDe cette façon, il n'est pas nécessaire de télécharger la vidéo ou le son du flux. Économie de bande passante pour l'utilisateur. \nAfin de maintenir à jour le statut (EN LIGNE ou HORS LIGNE) des canaux, une connexion Websocket est établie, \nqui reçoit des informations des flux pour pouvoir mettre à jour l'état, o cantidad de espectadores actuales.",
+            "how_it_works": "Comment ça fonctionne",
+            "how_it_works_text": "Environ toutes les ~60 secondes, l'application envoie un événement \"minute regardée\" à la chaîne qui est actuellement en cours de visionnage - ce qui suffit pour faire avancer les drops. Notez que cela contourne complètement le besoin de télécharger la vidéo et le son du flux réel. Pour maintenir à jour le statut (EN LIGNE ou HORS LIGNE) des chaînes, une connexion websocket est établie pour recevoir des événements sur les flux qui montent ou descendent, ou des mises à jour concernant le nombre actuel de spectateurs."
```

```suggestion
            "how_it_works_text": "Environ toutes les ~60 secondes, l'application envoie un événement \"minute regardée\" à la chaîne qui est actuellement en cours de visionnage - ce qui suffit pour faire avancer les drops. Notez que cela contourne complètement le besoin de télécharger la vidéo et le son du flux réel. Pour maintenir à jour le statut (EN LIGNE ou HORS LIGNE) des chaînes, une connexion websocket est établie pour recevoir des événements sur les flux qui montent ou descendent, ou des mises à jour concernant le nombre actuel de spectateurs.",
```

There's a missing comma at the end of L156, that's required for the translation to be valid JSON. You can verify the validity yourself here: https://jsonformatter.curiousconcept.com